setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='1.2.0',
+    version='2.0.0',
```

Comment:
I'm not really sure that this necessitates a 2.0.0 release as opposed to 1.3.0, but I don't actually have that strong of an opinion so carry on.